### ISSUE TITLE
electron-prebuilt on npmjs.com lists 0.32.1 as latest version

### ISSUE THREAD

**Issue by frankhale** [mafintosh/electron-prebuilt#56]
Hi,

I noticed that you have released 0.32.2 of electron-prebuilt but npmjs.com still lists 0.32.1 as the latest version.

**Comment by hankbao** [comment]
+1

**Comment by maxogden** [comment]
Hmm super weird... it should be automated and has worked all previous times. Must have been a hiccup this time. I just did it manually:

```
$  npm publish
npm ERR! registry error parsing json
+ electron-prebuilt@0.32.2
```

Took about 1 minute which is pretty weird, but is apparently published now.

**Comment by frankhale** [comment]
Awesome, thank you!

**Comment by seriema** [comment]
@maxogden wondering if this is happening again? Since `0.35.2` is available in 6697877 since yesterday.

https://www.npmjs.com/package/electron-prebuilt
```
maxogden maxogden published 4 days ago
0.34.4 is the latest of 55 releases
```

**Comment by maxogden** [comment]
@seriema thanks for pointing it out, found the cause and opened an issue here: https://github.com/johnmuhl/electron-prebuilt-updater/issues/4
